modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc

```diff
@@ -73,6 +73,17 @@ REMOVE n[$prop]
 
 | Added the ability to dynamically reference properties in xref:clauses/set.adoc#set-dynamically-a-property[SET] and xref:clauses/remove.adoc#remove-remove-a-property-dynamically[REMOVE] clauses.
 
+a|
+label:functionality[]
+label:new[]
+[source, cypher, role=noheader]
+----
+DROP [COMPOSITE] DATABASE ... [RESTRICT \| CASCADE ALIAS[ES]]
+----
+
+| Added the ability to drop any database aliases, that would otherwise block the deletion of the database, at the same time as the deletion of the database.
+This will affect local database aliases targeting the database and constituent database aliases belonging to the composite database.
```

Comment:
I think we can make it shorter? The fact that the capability was added suggests it wasn't possible before.
```suggestion
| Added the ability to drop database aliases while deleting a database.
This will affect local database aliases targeting the database and constituent database aliases belonging to the composite database.
For more information, see link:{neo4j-docs-base-uri}/operations-manual/{page-version}/database-administration/standard-databases/delete-databases/#delete-databases-with-aliases[Delete a database with local database aliases targeting it] and link:{neo4j-docs-base-uri}/operations-manual/{page-version}/database-administration/composite-databases/delete-composite-databases/#composite-databases-delete-with-aliases[Delete a composite database with constituent database aliases].
```